### DISCUSSION TITLE
Favorite entities overview

### DISCUSSION THREAD

**Discussion by dleding** [emsesp/EMS-ESP32#1911] [Ideas]
Hello to the ems-esp world,

how would it be to display the selected favorite entities of all system components as an integral overwiew?
Maybe this could render a nice startpage?

Of course, if integrated in i.e. Homeassistant this is easily done myself, so the priority of such a function is very minor.
Just, as i find me constantly changing between the different device views when fooling around eg. to get the different temperature values, this would be a very nice roundup of the favorite-marking functionality overall to my eyes.

Please tell your opinion!

**Comment by proddy** [comment]
I like the idea. Just a simple read-only dashboard (with no edit) is easy to build. It would only be for EMS device entities, as we don't have the favourite option for analog & temperature sensors and custom entities. 

What would you call it? Where should it be in the WebUI ?

**Reply by MichaelDvP** [reply]
We had a similar question in #1709, but that was to HA specific. 
But i was thinking about it and tested a small addition to add it to the dashboard as "device". 
The entity names should have a prefix with the devicename.
Unclear what to do with sensors, custom, etc. We don't have a favorite-flag for these. Always add or never add? An editable favorite-flag is a lot of work.
This is with only ems-entities:
![grafik](https://github.com/user-attachments/assets/01f70e5f-94dc-4b8c-abac-b6783add68be)

**Reply by proddy** [reply]
personally, I would call it Dashboard and have it above the menu options Devices & Sensors. It would only show favourite entities and all sensors that are enabled and have values. No edits and no click-throughs. Just keeping it simple for now. And this could be the default page when the app is loaded instead of /devices

**Reply by dleding** [reply]
> personally, I would call it Dashboard and have it above the menu options Devices & Sensors. It would only show favourite entities and all sensors that are enabled and have values. No edits and no click-throughs. Just keeping it simple for now. And this could be the default page when the app is loaded instead of /devices

Cant describe it any better...that way around would be optimal to my eyes.

**Reply by proddy** [reply]
Coming soon in dev.v44 - https://github.com/emsesp/EMS-ESP32/issues/1958
